hack/lib/cleanup.sh

```diff
@@ -79,14 +79,10 @@ readonly -f os::cleanup::dump_etcd
 function os::cleanup::internal::dump_etcd_v3() {
 	local full_url="${API_SCHEME}://${API_HOST}:${ETCD_PORT}"
 
-	local etcd_client_cert="${MASTER_CONFIG_DIR}/master.etcd-client.crt"
-	local etcd_client_key="${MASTER_CONFIG_DIR}/master.etcd-client.key"
-	local ca_bundle="${MASTER_CONFIG_DIR}/ca-bundle.crt"
-
 	os::util::ensure::built_binary_exists 'etcdhelper' >&2
 
-	etcdhelper --cert "${etcd_client_cert}" --key "${etcd_client_key}" \
-	           --cacert "${ca_bundle}" --endpoint "${full_url}" dump
+	etcdhelper --cert "${ETCD_CLIENT_CERT}" --key "${ETCD_CLIENT_KEY}" \
```

Comment:
Update the comment above if you change the API here.

What happens if these are unset?

Reply:
Updated comment above to include new vars. If they are not set, this part of cleanup will fail. I believe these vars not being set was causing an extended_clusterup failure at some point